template.yaml

```diff
@@ -380,6 +563,55 @@ Resources:
         AppComponent: "trapheus:Lambda"
         AppFunction: "trapheus:RestoreRDSInTargetAccountsAndRegions"
 
+  RestoreRDSClusterInTargetAccountsAndRegionsLambdaFunction:
```

Comment:
Can we check and move newer lambdas to graviton for cost optimisation  ,refer https://github.com/intuit/Trapheus/pull/167/files